cantools/db/formats/kcd.py

```diff
@@ -132,11 +135,20 @@ def _load_message_element(message, bus_name):
 
     for signal in message.findall('ns:Signal', NAMESPACES):
         signals.append(_load_signal_element(signal))
+        
+    if length == 'auto':
+        if signals:
+            last_signal = sorted(signals, key=lambda s: s.start)[-1]
+            length = math.ceil((last_signal.start + last_signal.length) / 8.0)
```

Comment:
I suggest replacing floating point calculation with integer calculations, as `length = ((last_signal.start + last_signal.length + 7) // 8)`. Floating point calculation is probably not an issue in this case, but sometimes rounding errors occurs.